plugins/copr.py

```diff
@@ -254,6 +269,15 @@ def _remove_repo(cls, repo_filename):
             raise dnf.exceptions.Error(str(e))
 
     @classmethod
+    def _disable_repo(cls, copr_username, copr_projectname):
+        exit_code = call(["dnf", "config-manager", "--set-disabled",
```

Comment:
> IMHO would be better to provide simple way to call other dnf subcommands from plugin.

I'm against this idea. At least in a form of command lines. I'd be OK if we publish the command classes and change their interface. But using something like ".find_command('foo').run('arg', 'arg')" is not acceptable to me.